### ISSUE TITLE
Most perks in this game are literal skill issue

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#25]
I rarely change up my perks, as per my... *rather peculiar* playstyle. But lately i've been doing it more often, and the thing i realized is, just how much these perks smell of skill issue.

Kevlar Armor - "i get hit too often"
Sprinter & Under Pressure - "i cannot dodge properly and need extra speed"
Quick Hands & Overkill - "i missed the entire mag in my primary gun, i need to shoot more bullets NOW"
Extended Mags & Quick Load Mags - "i am gonna hit 3 bullets having fired my entire normal mag, and it takes 6 bullets to kill"
Ghost - "i cannot properly deal with people so i am gonna camp instead"

That's, like, more than half of all perks.

Idk what to do about it. But i don't like it.

**Comment by TheGoldenGemXD** [comment]
Personally I think perks are fine the way they are, as the entire point of the perks is to enhance gameplay and assist players with any flaws they have.

**Comment by deferred-impact** [comment]
I am not denying that is the point of (most) perks, what i would like to discuss is whether that *should* be the point of (most) perks.

We do have skillful perks. Look at kill regen and express ammo. These perks inherently reward good aim, and if you don't hit your shots they are useless. Why not make all perks that way?

**Comment by Gregory-Y** [comment]
I agree. Extended mags in particular are very problematic because it just encourages bullet spam. Ghost isn't too much of a problem, but in the hands of someone who knows what they're doing yes they'll be a nightmare to deal with. Kevlar and sprup were already discussed in other issues i think.

I'll try to think of a solution to exmags in another issue.

**Comment by snipergamingop** [comment]
> I agree. Extended mags in particular are very problematic because it just encourages bullet spam. Ghost isn't too much of a problem, but in the hands of someone who knows what they're doing yes they'll be a nightmare to deal with. Kevlar and sprup were already discussed in other issues i think.
> 
> I'll try to think of a solution to exmags in another issue.

How about make exmag gives 25% more clip size? It would kind of balance it somehow since you can't spam 100 bullet out of M249 in one go anymore

**Comment by snipergamingop** [comment]
Also give sprinter a condition: if you get a kill you get 15% speed boost for 2s. That way it would be skill-based for chain kills/escaping sticky situations
